test/bench/ok/heap-32.drun-run.ok

```diff
@@ -1,5 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-debug.print: (50_227, +91_377_860, 1_446_975_760)
-debug.print: (50_070, +102_586_000, 1_520_068_314)
+debug.print: (50_227, +91_377_860, 1_437_185_617)
+debug.print: (50_070, +102_586_000, 1_509_442_237)
```

Comment:
Neat idea! But it would feel safer if the numbers wouldn’t change :-). Should they change?

Reply:
They should. Less cycles are used for allocation (closures).

Reply:
Oh, that’s cycles! Then all is well.